README.md

```
@@ -105,7 +105,8 @@ export { app };
 
 - **Generated Graph Layout**: 
 	- No mid-graph direction changes.
-    - No way to set node execution priority (n8n runs nodes top-to-bottom)
+	- No way to set node execution priority (n8n runs nodes top-to-bottom)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)